keps/693-multikueue/README.md

```diff
@@ -0,0 +1,240 @@
+
+## Proposal
+
+Introduce MultiKueue AdmissionCheck, controller and configuration API. 
+
+Establish the need for a designated management cluster.
+
+![Architecture](arch.png "Architecture")
+
+For each workload coming to a ClusterQueue (with the MultiKueue AdmissonCheck enabled)
+in the management cluster, and getting past the preadmission phase in the 
+two-phase admission process (meaning that the global quota is ok), 
+MultiKueue controller will clone it in the defined worker clusters and wait 
+until some Kueue running there admits the workload.
+If a remote workload is admitted first, the job will be created 
+in the remote cluster with a `kueue.x-k8s.io/prebuilt-workload-name` label pointing to that clone.
+Then it will remove the workloads from the remaining worker clusters and allow the
+single instance of the job to proceed. The workload will be also admitted in 
+the management cluster.
+
+There will be no job controllers running in the management clusters or they will be
+disabled for the workloads coming to MultiKueue-enabled cluster queues via annotation
+or some other, yet to be decided, mechanism. By disabling we mean that the controller
+will do no action on the selected objects, no pods (or other objects created and 
+allowing other controllets to update their status as they see fit.
+
+There will be just CRD/job definitions deployed. MultiKueue controller will copy the status
+of the job from the worker clusters, so that it will appear that the job
+is running inside the management clusters. However, as there is no job controller,
+no pods will be created in the management cluster. No controller will also overwrite
+the status that will be copied by the MultiKueue controller. 
+
+If the job, for whatever reason, is suspended or deleted in the management cluster,
+it will be deleted from the worker cluster. Deletion/suspension of the job 
+only in worker cluster will trigger the global job requeuing.
+Once the job finishes in the worker cluster, the job will also 
+finish in the management cluster.
+
+### User Stories (Optional)
+
+#### Story 1
+As a Kueue user I have clusters on different cloud providers and on-prem. 
+I would like to run computation-heavy jobs across all of them, wherever 
+I have free resources.
+ 
+#### Story 2
+As a Kueue user I have clusters in multiple regions of the same cloud 
+provider. I would like to run workloads that require the newest GPUs,
+whose on-demand availability is very volatile. The GPUs are available
+at random times at random regions. I want to use ProvisioningRequest
+to try to catch them.
+
+### Risks and Mitigations
+* Disabling the Job controller for all (or selected objects) may be problematic
+on environments where access to the master configuration is limited (like GKE).
+We are in talks with SIG-Apps to establish an acceptable way of using a non
+default controller (or none at all).
+
+* Etcd may not provide enough performance (writes/s) to handle very large 
```

Comment:
nit: etcd should always be lower case ^_^

Reply:
Done.